tests/integrations/raftstore/test_unsafe_recovery.rs

```diff
@@ -1215,9 +1231,10 @@ fn test_unsafe_recovery_has_commit_merge() {
     assert!(has_commit_merge);
 }
 
-#[test]
+#[test_case(test_raftstore::new_node_cluster)]
+// #[test_case(test_raftstore_v2::new_node_cluster)]
```

Comment:
why comment out

Reply:
Will enable it later, for now demote is not supported yet.